LFS-180 As a user, I can record radiotherapy treatment details

prabnur opened the pull request.

prabnur committed:
LFS-180: As a user, I can record radiotherapy treatment details
Initial version

prabnur committed:
LFS-180: As a user, I can record radiotherapy treatment details
Refactor JSON file

sdumitriu committed:
LFS-180: As a user, I can record radiotherapy treatment details
Fixed minAnswers instead of maxAnswers

sdumitriu committed:
LFS-180: As a user, I can record radiotherapy treatment details
Removed units and answer types from questions

marta- committed:
LFS-180 As a user, I can record radiotherapy treatment details
- Renamed question field ids
- Changed question type from text to boolean where appropriate
- Removed NA answer options (conditional display must be implemented to eliminate the need for NA answers)
- Removed unnecessary displayType where it matched the default
- Specified unitOfMeasurement where applicable

marta- reviewed (approved):
Works fine after the latest fixes.

sdumitriu commented on questionnaires/clinical-data/src/main/resources/SLING-INF/content/Radiotherapy.json:
```diff
@@ -0,0 +1,164 @@
+    "text": "Beam energy",
+    "minAnswers": 0,
+    "maxAnswers": 1,
+    "dataType": "text"
```
Ideally, this should be a number, measured in `MV`.

sdumitriu commented on questionnaires/clinical-data/src/main/resources/SLING-INF/content/Radiotherapy.json:
```diff
@@ -0,0 +1,164 @@
+    "text": "Boost beam energy",
+    "minAnswers": 0,
+    "maxAnswers": 1,
+    "dataType": "text"
```
Ideally, this should also be a number, measured in `MV`.

marta- committed:
LFS-180: As a user, I can record radiotherapy treatment details
[Addressing review comments]
- Added min value to numeric fields
- Changed "beam energy" dataType from "text" to "decimal" and added unit of measurement
